x-pack/plugins/ml/public/application/components/collapsible_panel/collapsible_panel.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSplitPanel,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import React, { type FC } from 'react';
+import { css } from '@emotion/react';
```

Comment:
This import was broken in the original PR. For some reason my IDE used `import { css } from '@emotion/react/dist/emotion-react.cjs';` path 😬